golang/cosmos/app/app.go

```diff
@@ -350,6 +361,10 @@ func NewAgoricApp(
 		tkeys:             tkeys,
 		memKeys:           memKeys,
 	}
+	// The VM is entitled to full awareness of runtime configuration.
```

Comment:
Can you explain here what it means for `resolvedConfig` to be "subject to consensus" as described in `chain-main.js`?

IIUC, `appOpts` can contain outside-of-consensus validator configuration, just one example of which is `minimum-gas-prices`, as well as several others set by `agd start` command line flags or the files in `~/.agoric/config/*.toml`.

I think what this PR actually does is a good thing (by including more inside- and outside-of-consensus parameters for cosmic-swingset to act on); I am just confused by the comments that seem to be implying something stricter than what I see implemented.

Reply:
`resolvedConfig` should definitely *not* have been included in `bootMsg`, that was a bug on my part. Fixed now, which I hope resolves your misunderstanding.